protocol_test.go

```diff
@@ -265,6 +265,8 @@ func TestReadHeaderTimeoutIsReset(t *testing.T) {
 // we expect the actual address and port to be returned,
 // rather than the ProxyHeader we defined.
 func TestReadHeaderTimeoutIsEmpty(t *testing.T) {
+	DefaultReadHeaderTimeout = 200 * time.Millisecond
```

Comment:
I think this may have impact on subsequent tests, so maybe override the listener timeout instead.

Reply:
The point of the test is to check the default.